stormpath-export

```diff
@@ -40,16 +39,16 @@ class StormpathExport(object):
 
     EXPORTS = ['tenants', 'applications', 'directories', 'groups', 'organizations', 'accounts']
 
-    def __init__(self, base_url=None):
```

Comment:
I think we also need to leave this in -- this way if someone specifies a base url via the command line, we can pass it into this class and use it during the setup of the client.